drivers/volume/kdmp/kdmp.go

```diff
@@ -765,6 +787,12 @@ func (k *kdmp) StartRestore(
 		dataExport.Annotations[utils.SkipResourceAnnotation] = "true"
 		dataExport.Annotations[utils.BackupObjectUIDKey] = backupUID
 		dataExport.Annotations[pvcUIDKey] = bkpvInfo.PersistentVolumeClaimUID
+		logrus.Info("enableKopiaExecutorDebugMode calling")
+		if enableKopiaExecutorDebugMode() {
+			logrus.Info("enableKopiaExecutorDebugMode able to find the debug mode present in kdmp-config cm")
+			dataExport.Annotations[utils.KopiaDebugModeEnabled] = "true"
+			logrus.Info("enableKopiaDebugMode annotation added in the CR")
```

Comment:
Same as above, if you are already checking checkFunction you can avoid this. Or you can remove the log.info in checkfunction() if you are printing here.. Too much of log messages will create high logfile payload.